keyform/static/keyform/js/add_form.js

```diff
@@ -12,4 +12,14 @@ $(document).ready(function() {
             $('#id_keydata_set-TOTAL_FORMS').val(parseInt(form_idx) - 1);
         }
     });
+
+    function mailbox_field() {
```

Comment:
We should probably pick a better name than `mailbox_field()` for this since it could technically hide the Core # field for **any** KeyType in the database, not just Mailbox (even though it will probably only be the Mailbox field in practice).